tests/Entity/Parameters/ParametersSerializerTest.php

```diff
@@ -23,24 +17,92 @@ protected function setUp()
         $this->serializer = $this->jmsSerializer();
     }
 
+    /**
+     * @test
+     */
+    public function it_serialises_nested_conditions()
+    {
+        $parameters = Parameters::findParameters(
+            Conditions::or(
+                Conditions::and(
+                    Conditions::eq('field1', 'value1'),
+                    Conditions::eq('field2', 'value2')
+                ),
+                Conditions::and(
+                    Conditions::eq('field3', 'value3'),
+                    Conditions::eq('field4', 'value4')
+                )
+            )
+        );
+        $expectedXml = <<<XML
+<?xml version="1.0" encoding="UTF-8"?>
+<parameters>
+  <conditions>
+    <or>
+      <condition>
```

Comment:
Wydaje mi się, że prawidłowa serializacja nie powinna mieć zagnieżdżonego `<condition>` - `<condition>` powinny mieć tylko pojedyncze warunki ("liście"). Choć to teoretyczne rozważania, bo API **chyba** nie przyjmuje żadnego zagnieżdżania.